### ISSUE TITLE
A11y_AzureToolsStorageExplorer_AddCORSRule_Keyboard: Loss of information is observed on the screen.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#1967]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp; #MAS1.4.10; #Keyboard; #Mac;   

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and Select "Configure CORS Settings" and hit enter.
5. Navigate to "Add" control and hit enter, "Add CORS Rule" screen gets opened.
6. Verify loss of information occurs on the screen.
7. "Max Age(in seconds) editable spin button" is not visible on the screen and by using Tab key keyboard focus goes to spin button but not spin button not visible on the screen.

**​Actual:​**
When screen gets opened Loss of information is observed on the screen.
* "Max Age(in seconds) editable spin button" is not visible on the screen and by using Tab key keyboard focus goes to spin button but spin button not visible on the screen, but the spin button is visible on the screen by Mouse using Vertical scrolling control.

**Expected:**
When screen gets opened Loss of information should not be observed on the screen.
* "Max Age(in seconds) editable spin button" should be visible on the screen without using or requiring Vertical scrolling control.

**UserImpact:**
Keyboard only users may not be able to know about entire content present on the screen if loss of information is observed.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BE12828DF-B35C-4F4D-9289-1F932E19F8CF%7D&file=MAS%201.4.10%20%E2%80%93%20Reflow.docx&action=default&mobileredirect=true&cid=7ce76489-c95d-4139-8454-2216135f7470

**Attachment for Reference:**
[1967_A11y_AzureToolsStorageExplorer_AddCORSRule_Keyboard.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669408/1967_A11y_AzureToolsStorageExplorer_AddCORSRule_Keyboard.pptx)

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0, this issue is no longer reproduced. Hence Closing the bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
